src/main/java/io/mosip/iiitb/odte/config/OnDemandAppConfig.java

```diff
@@ -0,0 +1,103 @@
+package io.mosip.iiitb.odte.config;
+
+import org.aeonbits.owner.Config;
+
+
+@Config.LoadPolicy(Config.LoadType.MERGE)
+@Config.Sources({
+        "${runtimeOndemandAppConfigPropertiesPath}",
+        "system:properties",
+        "system:env",
+        "classpath:ondemand-default.properties"
+})
+public interface OnDemandAppConfig extends Config {
+    @Key("mosip.iiitb.ondemand.db.jdbc.url")
+    String dbUrl();
+
+
+    @Key("mosip.iiitb.ondemand.db.jdbc.user")
+    String dbUsername();
+
+    @Key("mosip.iiitb.ondemand.db.jdbc.password")
+    String dbPassword();
+
+    @Key("mosip.iiitb.ondemand.db.name")
+    String dbName();
+
+    @Config.Key("mosip.iiitb.ondemand.kafka.groupId")
+    String kafkaGroupId();
+
+    @Config.Key("mosip.iiitb.ondemand.kafka.brokers")
+    String kafkaBrokers();
+
+    @Config.Key("mosip.iiitb.ondemand.kafka.topic")
+    String kafkaTopic();
+
+
+    @Config.Key("mosip.iiitb.ondemand.kafka.port")
```

Comment:
do we need the iiitb naming at the property level ?

Reply:
In my mind, the configuration options should ideally match with `<package-name>.some.option`. So that when there is a remote file specifying configuration, it's explicit for which service this configuration is meant for. I admit I have not followed that practice consistently since we kept changing the package name a lot. But do let me know why you think it shouldn't be same namespaced as the service package name.